resources/crds/ripsaw_v1alpha1_ycsb_cr.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: ycsb-couchbase-benchmark
   namespace: my-ripsaw
 spec:
+  elasticsearch:
+    server: http://my.elasticsearch.server:80
+    index_name: ripsaw-ycsb
```

Comment:
Why add this here? it is defaulted to the same? We should standardize the crs w/ or w/o  them

Reply:
Indeed, I added it to standarize examples, also I removed index_name, which is not present in other examples